lib/elasticsearch.ts

```diff
@@ -41,3 +51,29 @@ export function get_indices() {
     return filtered_indices.sort();
   });
 }
+
+// We want to keep 14 days of indices. Let's do 15 so I don't have to think hard about what happens
+// when the time is in UTC or when we're before/after midnight.
+const NUM_DAYS = 15;
```

Comment:
I think this can stay 14 with the idea that at any given time there there should be 14 indexes, and occasionally we will have 15 until this action is run.

Also, what about making this configurable?